src/pydocstyle/parser.py

```diff
@@ -218,6 +218,14 @@ def is_overload(self):
                 return True
         return False
 
+    @property
+    def is_property(self):
+        """Return True iff the method decorated with property."""
+        for decorator in self.decorators:
+            if "property" in decorator.name:
```

Comment:
Pretty sure if I create a decorator named `myproperty` this will disable D401 which is far from ideal. Why not `if decorator.name == "property":`?